TOPdeskPS/functions/Supporting Files/Get-TdSupplierContact.ps1

```diff
@@ -0,0 +1,70 @@
+﻿function Get-TdSupplierContact {
+    <#
+.SYNOPSIS
+    Gets list of supplier  contacts
+.DESCRIPTION
+    Returns list of supplier contacts
+    .PARAMETER Name
+    retrieve only suppliers with names starting with this string
+    .PARAMETER PageSize
+    The amount of suppliers to be returned per request. Must be between 1 and 10000, default is 1000.
+    .PARAMETER Archived
+        Whether to retrieve archived incidents.
+    .PARAMETER PageSize
+		The amount of incidents to be returned per request. The default value is 10 and the maximum value is 100.
+	.PARAMETER Start
+		This is the offset at which you want to start listing suppliers at.
+        The default value is 0.
+    .PARAMETER SupplierId
+    ID of the Supplier. See Get-TdSupplier
+    .EXAMPLE
+    PS C:\> Get-TdSuppliercontact
+    Returns list of supplier contacts
+      .EXAMPLE
+    PS C:\> Get-TdSupplier | Get-TdSupplierContact
+    Returns list of
```

Comment:
update help

Reply:
this one needs some more look it doesnt seem to match the API at all?

Reply:
added $name param, because it's part of the code, but the API does not have a name param

Reply:
yeah i think we should redo it as 2 commands. Get-TdSupplierContact and Get-TdSupplierContactDetail